tools/gbench/report.py

```diff
@@ -7,7 +7,9 @@
 import copy
 import random
 
-from scipy.stats import mannwhitneyu
+from scipy.stats import mannwhitneyu, gmean
+from numpy import array
+from pandas import Timedelta
```

Comment:
pandas should be added to `requirements.txt` i think.

Reply:
Thinking about it, should i just hardcode the multipliers needed?

Reply:
i have no issue with using pandas. i think it's cleaner to use something that is known and does the job with a well-known API.